client/my-sites/stats/feedback/index.tsx

```diff
@@ -1,18 +1,24 @@
+import { useState } from 'react';
+import FeedbackModal from './modal';
+
 import './style.scss';
 
-function StatsFeedbackCard() {
+const StatsFeedbackCard = () => {
```

Comment:
Do you have a preference for using anonymous functions assigned to const variables over standard function syntax, Dognose?

Just curious because personally, I'd prefer we not turn all our components into const pointers. It feels a bit fiddly to type and prevents function hoisting. I also recall reading that it might have performance implications due to how const pointers are memory-mapped vs standard functions but I'd have to dig deeper to see if that is true or not.

Reply:
Restored in [25be9c8](https://github.com/Automattic/wp-calypso/pull/93959/commits/25be9c81097ebc6fe1d9630269c97d3723914502). cc @a8ck3n